plugins/services/src/js/events/__tests__/SDKEndpointsActions-test.js

```diff
@@ -0,0 +1,157 @@
+const RequestUtil = require("mesosphere-shared-reactjs").RequestUtil;
```

Comment:
we actually can use `import` in tests as well

Reply:
Used the import, it is unfortunate that most of the tests use the require.

Reply:
Yeah, we should update all of the test till then let's please be consistent with at least the resp. package/plugin.

Reply:
changed back

Reply:
We could just do one by one, esp. for all new files.

Reply:
It's super easy to change them all at once. I'll file a JIRA and schedule for next sprint.